### DISCUSSION TITLE
"Line must be part of diff" in PR - isn't desirable

### DISCUSSION THREAD

**Discussion by ghost** [community/community#5900] [Pull Requests]
Reviewing a pull request today, I ran into the error: "Line must be part of diff" when trying to add a comment.

This in my opinion is an undesired requirement for comments. Two pointst:

1. When commenting on PRs I do usually look at the code around the changes too, and do ask for people to fix minor issues there, and I kind of expect people to look through things like compiler/linting warnings, and the code close to what they are working on. It doesn't really matter who overlooked the issue originally, it matters that it gets fixed

2. There might be relevant code for the task at hand that the developer have overlooked and I want highlighted in the response to PR.

**Comment by acgoldma** [comment]
Possible duplicate of #4452
